packages/graphiql/src/utility/onHasCompletion.js

```diff
@@ -15,76 +15,78 @@ const md = new MD();
  * about the type and description for the selected context.
  */
 export default function onHasCompletion(cm, data, onHintInformationRender) {
-  const CodeMirror = require('codemirror');
+  import('codemirror')
+    .catch(console.err)
```

Comment:
why?

Reply:
Because otherwise the error function isn’t bound to console, and this can cause trouble under certain circumstances (particularly when console has been replaced).

Reply:
(Also errors are unhandled.)